construction/data/res_config_settings.xml

```diff
@@ -1,23 +1,133 @@
 <?xml version='1.0' encoding='UTF-8'?>
-<odoo noupdate="1">
-    <record id="res_config_account_setting" model="res.config.settings">
-        <field name="group_project_stages" eval="1"/>
-        <field name="group_discount_per_so_line" eval="1"/>
-        <field name="group_product_pricelist" eval="1"/>
-        <field name="group_warning_sale" eval="1"/>
-        <field name="group_warning_purchase" eval="1"/>
-        <field name="invoiced_timesheet" eval="'approved'"/> 
-        <field name="group_warning_stock" eval="1"/> 
-        <field name="group_stock_reception_report" eval="1"/> 
-        <field name="group_sale_delivery_address" eval="1"/>
-        <field name="documents_product_settings" eval="1"/>
-        <field name="documents_hr_settings" eval="1"/>
-        <field name="group_analytic_accounting" eval="1"/>
-        <field name="group_uom" eval="1"/>
-        <field name="group_sale_order_template" eval="1"/>
+<odoo>
+    <record id="x_setting_field_progress_management" model="ir.model.fields">
+        <field name="ttype">boolean</field>
+        <field name="field_description">Progress Management</field>
+        <field name="model_id" ref="base_setup.model_res_config_settings"/>
+        <field name="name">x_setting_progress_management</field>
```

Comment:
Looking twice, I think we should already use the same as standard logic for naming, i.e. x_module_technical_name
```suggestion
        <field name="name">x_module_construction_progress_management</field>
```
so that if we do the modif in standard, we don't have to change all names again